python/docs/examples.rst

```diff
@@ -0,0 +1,203 @@
+    >>> with dlisio.load(filename) as (f, *tail):
+    ...     f.describe()
+    ------------
+    Logical File
+    ------------
+    Description  : dlis(MSCT_200LTP)
+    Frames       : 2
+    Channels     : 104
+    Object count : 570
+
+    Known objects
+    --
+    FILE-HEADER             : 1
+    ORIGIN                  : 1
+    CALIBRATION-COEFFICIENT : 24
+    CHANNEL                 : 104
+    FRAME                   : 2
+
+    Unknown objects
+    --
+    440-CHANNEL        : 95
+    440-OP-CORE_TABLES : 250
+    440-OP-CHANNEL     : 93
+
+Accessing objects
+-----------------
+
+Think of :ref:`Logical files` as pools of objects with different types.  All
+objects of a type can be reached by name:
+
+.. code-block:: python
+
+    >>> for ch in f.channels
+    ...     _ = ch
+
+See :ref:`Logical files` for a full list of all object types.
+
+:py:func:`dlisio.dlis.object` let you access a specific object: 
+
+.. code-block:: python
+
+    >>> obj = f.object('CHANNEL', 'TDEP')
+
+Objects can also be searched for with :py:func:`dlisio.dlis.match()`:
+
+.. code-block:: python
+
+    >>> objs = f.match('T.*')
+
+Inspect an object with the :code:`.describe()`-method:
+
+.. code-block:: python
+
+    >>> obj.describe()
+    -----
+    Frame
+    -----
+    name   : 800T
+    origin : 2
+    copy   : 0
+
+    Channel indexing
+    --
+    Indexed by       : TIME
+    Interval         : [33354518, 35194520]
+    Direction        : INCREASING
+    Constant spacing : 800
+    Index channel    : Channel(TIME)
+
+    Channels
+    --
+    TIME TDEP ETIM LMVL UMVL CFLA OCD  RCMD RCPP CMRT
+    RCNU DCFL DFS  DZER RHMD HMRT RHV  RLSW MNU  S1CY
+    S2CY RSCU RSTS UCFL CARC CMDV CMPP CNU  HMDV HV
+    LSWI SCUR SSTA RCMP RHPP RRPP CMPR HPPR RPPV SMSC
+    CMCU HMCU CMLP
+
+Frames and Channels
+-------------------
+
+See :ref:`Curves` for information about the relationship between Channels and
+Frames. Have a look at :ref:`Channel` and :ref:`Frame`, they contain some
+useful metadata in addition to the curve-values! 
+
+Channels belonging to a Frame can be accessed directly through 
+:py:func:`dlisio.plumbing.Frame.channels`:
+
+.. code-block:: python
+    
+    >>> frame.channels[0]
+    Channel(TDEP)
+
+Likewise, the parent-frame of a Channel can be accessed through the channel:
+
+.. code-block:: python
+
+    >>> ch.frame
+    Frame(800T)
+
+The actual curve data of a Channel is accessed by :py:func:`dlisio.plumbing.Channel.curves()`,
+which returns a structured numpy array that support common slicing operations:
+
+.. code-block:: python
+    
+    >>> curve = ch.curves()
+    >>> curve[0:5]
+    array([852606., 852606., 852606., 852606., 852606.], dtype=float32)
+
+Access all curves in a frame with :py:func:`dlisio.plumbing.Frame.curves()`.
+The returned structured numpy array can be indexed by Channel mnemonics
+and/or sliced by samples: 
+
+.. code-block:: python
+    
+    >>> curves = fr.curves()
+    >>> curves[['TDEP', 'TENS_SL']][0:5]
```

Comment:
[ [ ] ] [ ] structure for this example may be too confusing. I from the first glance wouldn't be able to tell you what [ [ ] ] gives me